db/migrate/20220326181609_posts.rb

```diff
@@ -0,0 +1,12 @@
+class Posts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :Posts do |t|
+      t.references :User, null: false, foreign_key: true, index: true
+      t.string :Title
+      t.text :Text
+      t.timestamps
+      t.integer :CommentsCounter, default: 0, null: false
+      t.integer :LikesCounter, default: 0, null: false
```

Comment:
- [ ] <b>REQUIRED:</b> The same thing goes here, Please use `lower case` and `snake case`.